### PR TITLE
[MULTIARCH-1973] 4.10 installation doc updates from IBM P

### PR DESCRIPTION
**Pull request by alishaIBM**
- OCP version for cherry-picking: enterprise-4.10

- Jira : https://issues.redhat.com/browse/MULTIARCH-1973

- Google doc : https://docs.google.com/document/d/1PomAqIMBORc1_XyJwM_zy6V8EieWiM0UOJG_1brcqTo/

- Preview : https://deploy-preview-41819--osdocs.netlify.app/openshift-enterprise/latest/welcome/index.html

- QE review IBM P : Manoj Kumar , Julie Mathew

### CODE REVIEW

File: installing/installing_ibm_power/installing-ibm-power.adoc [hunk 1]
```diff
@@ -110,8 +110,6 @@ include::modules/installation-registry-storage-non-production.adoc[leveloffset=+
 
 include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
 
-include::modules/cluster-telemetry.adoc[leveloffset=+1]
-
 .Additional resources
```
Comment (SNiemann15) [inline installing/installing_ibm_power/installing-ibm-power.adoc:113]: Also remove lines 113 - 115 they refer to Telemetry

File: installing/installing_ibm_power/installing-ibm-power.adoc [hunk 2]
```diff
@@ -110,12 +110,6 @@ include::modules/installation-registry-storage-non-production.adoc[leveloffset=+
 
 include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
 
-include::modules/cluster-telemetry.adoc[leveloffset=+1]
```
Comment (SNiemann15) [inline installing/installing_ibm_power/installing-ibm-power.adoc:113]: Add this again and in addition insert a new line [role="_additional-resources"] before line 115. 

Telemetry only needs to be removed for restricted install  installing\installing_ibm_power\installing-restricted-networks-ibm-power.adoc

**Comment by lsmcfadden** [comment]
@SNiemann15 @alishaIBM do we need to add Julie and Manoj here to sign off?

**Review by lpettyjo — COMMENTED** [review]
LGTM

**Pull request merged**